UnitTestingApp.js

```diff
@@ -8,19 +8,27 @@
  * Class for running unit tests
  */
 
- let UnitTestingApp = (function () {
+let UnitTestingApp = (function () {
 
+  _nTests = 0;      // Total number of tests executed
+  _nFailTests = 0;  // Total test failed
+  _nPassTests = 0;  // Total test passed
+  _levelInfo = 1;   // Level of information to show in the output (0-summary, 1-trace and test result information)
```

Comment:
Do these need to be here at all?

Reply:
per my last comment, I have modified the source code, waiting for your response to update it. I modified this using WeakMap.